SDL_stbimage.h

```diff
@@ -200,6 +200,7 @@ SDL_STBIMG_DEF SDL_bool STBIMG_stbi_callback_from_RW(SDL_RWops* src, STBIMG_stbi
 #endif // 0 (STBIMG_stbi_callback_from_RW() example)
 
 
+#if SDL_MAJOR_VERSION > 1
```

Comment:
doesn't SDL1 also have SDL_RWops?

Reply:
Yes, but it doesn't have [`SDL_RWsize()`](https://wiki.libsdl.org/SDL_RWsize).

> This function is available since SDL 2.0.0.

Reply:
ok